draft-colitti-ipsecme-esp-ping.mkd

```diff
@@ -74,8 +75,74 @@ An IPv6 node that desires to determine whether the path to a particular destinat
 
 If the destination supports ESP, and wishes to reveal to the sender that it does so, it SHOULD reply with an ESP Echo Reply packet. ESP Echo Reply packets are ESP packets with an SPI value of (8-TBD), a Next Header value of 59, and no payload.
 
+The ESP Echo Request and Reply packets utilize the standard ESP packet format as described in Section 2 of {{!RFC4303}} with the following changes:
+
+* SPI set to
+  * [ESP-ECHO-REQUEST] for ESP Echo Request
+  * [ESP-ECHO-REPLY] for ESP Echo Reply
+* The Next Header field of the ESP header SHOULD be set to 59 (No Next Header).
+* No Integrity Check Value-ICV.
+
+The payload has the following format:
+
+     0                   1                   2                   3
+      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     |        ECHO Identifier        |      ECHO Sequence Number     |
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     |     Data ...                                                  |
+     +-+-+-+-+-
+
+
+    * ECHO Identifier: An identifier to aid in matching Echo Replies to Echo Requests. May be zero.
+
+    * ECHO Sequence Number: An identifier to aid in matching Echo Replies to Echo Requests. May be zero.
```

Comment:
May or MAY?

Reply:
https://datatracker.ietf.org/doc/html/rfc4443#section-4 uses lower case. I do not have any preference here, will change to MAY...